redis.conf

```diff
@@ -518,8 +518,9 @@ dir ./
 #    still reply to client requests, possibly with out of date data, or the
 #    data set may just be empty if this is the first synchronization.
 #
-# 2) If replica-serve-stale-data is set to 'no' the replica will reply with
-#    an error "SYNC with master in progress" to all commands except:
+# 2) If replica-serve-stale-data is set to 'no' the replica will reply with error
+#    "MASTERDOWN Link with MASTER is down and replica-serve-stale-data is set to 'no'"
+#    to all data access commands, excusing commands such as :
```

Comment:
excusing -> excluding

Reply:
:disappointed: 3 hours too late.. care to make another PR?

Reply:
Sorry! :-) It's not important.

Reply:
I added it in a old PR... https://github.com/redis/redis/pull/9153 (both are simple small cleanups)